Header header

float32 velocity       # the velocity of the car in [m/s]
